test/persistence-hooks.suite.js

```diff
@@ -3016,12 +3016,12 @@ module.exports = function(dataSource, should, connectorCapabilities) {
         monitorHookExecution(['access', 'before save']);
 
         TestModel.observe('access', function(ctx, next) {
-          ctx.query = { where: { id: { neq: existingInstance.id }}};
+          ctx.query = { where: { id: { eq: existingInstance.id }}};
```

Comment:
This is changing the intent of the test. Before, the query did not match the existing instance, I presume a new record was created? Now the existing record is updated. Is that ok?